src/test/isolation2/sql/resgroup/resgroup_functions.sql

```diff
@@ -8,4 +8,10 @@ INSERT INTO resgroup_function_test(groupid, num_running, num_queueing, num_queue
 SELECT s.groupid, s.num_running, s.num_queueing, s.num_queued, s.num_executed
 FROM pg_resgroup_get_status(NULL::oid) s(groupid, num_running, num_queueing, num_queued, num_executed, total_queue_duration, cpu_usage, memory_usage) LIMIT 1;
 
-SELECT count(num_executed)>0 FROM resgroup_function_test WHERE num_executed IS NOT NULL;
```

Comment:
@andr-sokolov, can you explain why existing test query is redundant?

Reply:
I have added insert into regroup_function_test so that this select becomes necessary again

> @andr-sokolov, can you explain why existing test (checking of ~num_executed~ lines) query is redundant? Do you rely on `INSERT 2` output?